.github/workflows/test.yml

```diff
@@ -22,15 +22,19 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - run: rustup target add wasm32-unknown-unknown
       - name: Test
-        run: cargo test --all --features unstable
+        env:
+          RUSTFLAGS: -D warnings
```

Comment:
I'd put this to top-level, seems right to just blanket-apply it everywhere:

https://github.com/rust-lang/rust-analyzer/blob/master/.github/workflows/ci.yaml#L12-L18